snowflake/network-observability.md

```diff
@@ -0,0 +1,78 @@
+# Network Observability
+
+We use Grafana as the tool to query data from snowflake, and visualize the
+networking flows in the cluster(s).
+
+## Deployment
+
+We suggest install and run Grafana using the official Docker image.
+
+Before running Grafana, we will first download the snowflake datasource plugin.
+We suggest create your own plugin directory.
+
+```bash
+mkdir your-plugin-path && cd your-plugin-path
+wget https://github.com/michelin/snowflake-grafana-datasource/releases/download/v1.2.0/snowflake-grafana-datasource.zip
+unzip snowflake-grafana-datasource.zip
+```
+
+Then run Grafana with Docker:
+
+```bash
+docker run -d \
```

Comment:
is there a way to load the JSON dashboard files as well when you start Grafana (rather than having to import them 1 by 1 later)?

Reply:
done

In order to provision dashboards, we also need to provision datasource. For now, both dashboards and datasource are provisioned. Users are expected to configure datasource in a yaml file.